int32 index
int32 path_index
geometry_msgs/Pose pose
float64 velocity
float64 heading
float64 curvature
int32 mission_state
